### PR TITLE
updated current team and added 2022 batch in alumini section.

### PR DESCRIPTION
**Pull request by Dhruv454000**
- Updated current team
- Added 2022 batch in alumini section

### CODE REVIEW

**Comment by amanchhaparia** [comment]
@Dhruv454000, It would be better if you issue different PRs for wall-e/mario bot photo, updating the notification message. This helps to track the PRs better incase of any conflicts.
 If possible add the photo of mario bot black version instead of this transparent body.

**Comment by Dhruv454000** [comment]
> @Dhruv454000, It would be better if you issue different PRs for wall-e/mario bot photo, updating the notification message. This helps to track the PRs better incase of any conflicts. If possible add the photo of mario bot black version instead of this transparent body.

Done

**Comment by Dhruv454000** [comment]
> @Dhruv454000, It would be better if you issue different PRs for wall-e/mario bot photo, updating the notification message. This helps to track the PRs better incase of any conflicts. If possible add the photo of mario bot black version instead of this transparent body.

Okay , I will issue new pr for notification part and walle, mario bot image updation.

File: data/senate.js [hunk 1]
```diff
@@ -1,109 +1,301 @@
...
+    name: 'Gaurav kumar ',
+    imgName: 'Team/2024/Gaurav _Kumar_24.jpg',
+    sub: <>B.Tech Textile ' 24</>,
+    linkedInLink: 'https://www.linkedin.com/in/gaurav-kumar-b7298220a/',
+    githubLink: 'https://github.com/GauravKumar9920',
+  },
+  {
+    name: 'Gautam Agrawal ',
+    imgName: 'Team/2023/Gautam_Agrawal.jpeg',
+    sub: <>B.Tech Information Technology ' 23</>,
```
Comment (kkothari2001) [inline data/senate.js:242]: All old positions are to be retained but with the right year eg "GS 2021"

**Comment by kkothari2001** [comment]
Hey, @Dhruv454000 I'm sorry if I was not clear enough. I meant post screenshots (of how the cosmetic changes make the site look) here

**Comment by kkothari2001** [comment]
Look at #54 for example

**Comment by Dhruv454000** [comment]
> Hey, @Dhruv454000 I'm sorry if I was not clear enough. I meant post screenshots (of how the cosmetic changes make the site look) here

sorry for that , This is how it looks

image

image

**Comment by Dhruv454000** [comment]
image


image


image

**Pull request merged**